src/__tests__/category/__snapshots__/category.query.test.js.snap

```diff
@@ -0,0 +1,24 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Tests should not be in `src` directory, it should be on the same level as `src` folder. That's why we have `src` folder at all